api/authentication.py

```diff
@@ -95,6 +95,13 @@ def get_fxa_uid_from_oauth_token(token: str, use_cache: bool = True) -> str:
         raise NotFound("FXA did not return an FXA UID.")
     fxa_uid = str(raw_fxa_uid)
 
+    if "https://identity.mozilla.com/apps/relay" not in fxa_resp_data["json"].get(
+        "scope", ""
+    ):
+        raise AuthenticationFailed(
+            "FXA missing is missing scope: https://identity.mozilla.com/apps/relay."
```

Comment:
suggestion (non-blocking):

```suggestion
            "FXA token is missing scope: https://identity.mozilla.com/apps/relay."
```